heltour/settings.py

```diff
@@ -196,8 +203,19 @@
 EMAIL_HOST = env("EMAIL_HOST", default="localhost")
 EMAIL_PORT = env("EMAIL_PORT")
 EMAIL_USE_TLS = env("EMAIL_USE_TLS")
-EMAIL_HOST_USER = env("EMAIL_HOST_USER", default="")
-EMAIL_HOST_PASSWORD = env("EMAIL_HOST_PASSWORD", default="")
+
+if env("EMAIL_SMTP_USER_FILE", "") and os.path.exists(env("EMAIL_SMTP_USER_FILE")):
```

Comment:
You could probably make a quick helper that does this for us and then re-use it throughout.